packages/components/src/Actions/ActionFile/ActionFile.component.js

```diff
@@ -1,7 +1,7 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import { OverlayTrigger } from 'react-bootstrap';
-import classNames from 'classnames';
+import classnames from 'classnames';
```

Comment:
just a change on character case could lead to a lot on misunderstanding maybe something more relevant as name ?